java/BUILD

```diff
@@ -23,12 +23,14 @@ cacerts_java(
         # cmd = ["/path/to/deploy.jar", "--option1", ...]
     ],
     symlinks = {
-        "/usr/bin/java": "/usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java",
+        "/usr/bin/java": java_executable_path,
     },
     tars = [":cacerts_java"],
-) for mode in [
-    "",
-    ":debug",
+) for (rule_name, jre_deb, java_executable_path) in [
```

Comment:
Probably just get some bazel person to look at this, is there a way to share the java_executable_path reference or would that make this section look super wonky?
@nlopezgi wdyt?

Reply:
I don't feel super strongly about this. I was concerned about each java path could potentially be shared for each java8, java11 option, but it seems like it's not worth the trouble right now